src/uniprotkb/config/UniProtKBColumnConfiguration.tsx

```diff
@@ -275,9 +275,7 @@ UniProtKBColumnConfiguration.set(UniProtKBColumn.organismHosts, {
     return (
       <ExpandableList descriptionString="hosts" displayNumberOfHiddenItems>
         {organismHosts?.map((host) => (
-          <p key={host.taxonId}>
-            <TaxonomyView key={host.taxonId} data={host} />
-          </p>
+          <TaxonomyView key={host.taxonId} data={host} />
```

Comment:
I see this bit of code is in src/shared/components/entry/TaxonomyView.tsx l.121 too, do you mind taking the same approach there too?

Reply:
Yep, I've used the ExpandandableList as that's what we do in other places in the entry page.